frontend/app/routes/_gcweb-app.apply.apply-now.tsx

```diff
@@ -0,0 +1,40 @@
+import type { LoaderFunctionArgs } from '@remix-run/node';
+
+import { useTranslation } from 'react-i18next';
+
+import { ButtonLink } from '~/components/buttons';
+import { getRaoidcService } from '~/services/raoidc-service.server';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('intake-forms');
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const raoidcService = await getRaoidcService();
+  await raoidcService.handleSessionValidation(request);
+
+  return '';
```

Comment:
**Suggestion/Nitpick:** Returning an empty string seems odd. An example from the Remix docs does this instead 
```javascript
export async function loader({ request }: LoaderFunctionArgs) {
  return json({ ok: true });
};
```

See https://remix.run/docs/en/main/route/loader

Reply:
done